src/Clash/Shake/Xilinx.hs

```diff
@@ -143,7 +146,7 @@ vivado fpga kit@ClashKit{..} outDir srcDir topName = do
                      , [ "project" .= T.pack projectName ]
                      , [ "top" .= T.pack topName ]
                      , targetMustache fpga
-                     , [ "board" .= T.pack "digilentinc.com:nexys-a7-50t:part0:1.0" ] -- TODO
```

Comment:
OK so I guess we want to have separate `FPGA` and `Board` components of the `Target`?

Reply:
I've pushed to `master` so that the Vivado backend is parameterized both by the FPGA part and the dev board descriptor. Can you rebase this on top of that?